GitUI/UserControls/BlameControl.cs

```diff
@@ -278,19 +278,19 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
                 }
                 else
                 {
-                    gutter.Append(line.Commit.AuthorTime.ToString(CultureInfo.CurrentCulture));
+                    gutter.Append(line.Commit.AuthorTime.ToString(CultureInfo.CurrentCulture.DateTimeFormat.ShortDatePattern));
```

Comment:
⚠️ This needs to be discussed. We do show dates and times everywhere else.
For now please remove it from this PR.

Besides we have a setting to show relative dates (_Settings > Appearance > General_), perhaps we should honour this settings here.

Reply:
I will not be very pleased if I have to remove it even if this not consistent with what is done everywhere else in the application because I think that miss the point of why this setting is added here. 

The goal of this PR was to give more flexibility to gain precious horizontal space (in case we need to display long lines on small screens). So the constraints are a little different that elsewhere in the application and I think it add value to behave a little differently.

I did it on purpose once I discovered that most of the times, displaying time in the blame is useless where date is enough.

The other reason is that I'm not very motivated to take some time to remove part of the code already written that add value 😢 It's just a setting that most will ignore if not interested...
If you **really** want to remove it, I will do it to make the other settings merged but it's not really what I want to end of doing...